common/wonder_upgrades/wc_wonder_upgrades.txt

```diff
@@ -140,6 +140,26 @@ upgrade_endless_corridors = {
 	owner_modifier = {
 	}
 }
+upgrade_construct_quarter = {
+	picture = GFX_upgrade_icon_skulls
+
+	potential = { }
+	allow = { }
+	active = { }
+
+	flags = {
+		is_singular
+	}
+	build_time = 40
+	gold_cost_flat = 470
+	gold_cost_ticking = 1
+	local_modifier = {
+		undead = 225
```

Comment:
Will give any holding in province a 225 special troops.
Make sure that scourge holdings have only undeads an no other special troops.